plugins/slack_app/app/models/slack_message.rb

```diff
@@ -0,0 +1,116 @@
+class SlackMessage
+  def initialize(deploy)
+    @deploy = deploy
+  end
+
+  def deliver
+    url = DeployResponseUrl.find_by(deploy_id: @deploy.id).try(:response_url)
```

Comment:
Only deploys triggered by the slash-command will have an entry in this table, so we should fail softly if we don't have a place to send the message. This seemed like a reasonable way to deal with that situation.